code/script.js

```diff
@@ -36,12 +44,183 @@ const greeting = () => {
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
+//Question 1
+
+const handleResponse = () => {
+  let userName = document.getElementById('name-input').value;
+  showMessage(`Nice to meet you ${userName}!`, 'bot')
+   
+}
+
+//Question 2
+
+const moodOptions = () => {
+  let userName = document.getElementById('user-input');
+  showMessage(`How are you feeling today?`, 'bot')
+  inputWrapper.innerHTML = `
+    <button id="option1">happy</button>
+    <button id="option2">sad</button>
+    <button id="option3">angry</button>
+  `
+  document.getElementById("option1").addEventListener("click", () => {
+    showMessage("I'm happy", "user")
+  })
+  document.getElementById("option2").addEventListener("click", () => {
+    showMessage("I'm sad", "user")
+  })
+  document.getElementById("option3").addEventListener("click", () => {
+    showMessage("I'm angry", "user")
+  })
+  
+  document.getElementById("option1").addEventListener("click", () => {
+    showMessage("I see you are feeling happy!😊 Here's a list that suits your mood. Go ahead and choose one! 👇", "bot")
+    inpt = () => { 
+      inputWrapper.innerHTML= `
+        <select id="select" name="songs">
+          <option value selected disabled>Select a song</option>
+          <option id="select">Happy by William Farrel</option>
+          <option id="select">Here comes the sun by the Beatles</option>
+          <option id="select">Shiny happy people by REM</option>
+        </select>
+      `
+      const select = document.getElementById("select");
+      let userName = document.getElementById('user-input');
+      select.addEventListener("change", () =>
+      greatChoice(select.value))
+    }
+  })
```

Comment:
You already created an event listener for this event and DOM object. You should clean up it a little. 😸